src/main/java/de/hysky/skyblocker/skyblock/item/slottext/adders/HotmPerkLevelAdder.java

```diff
@@ -0,0 +1,49 @@
+package de.hysky.skyblocker.skyblock.item.slottext.adders;
+
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import de.hysky.skyblocker.skyblock.item.slottext.SimpleSlotTextAdder;
+import de.hysky.skyblocker.skyblock.item.slottext.SlotText;
+import de.hysky.skyblocker.utils.ItemUtils;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+import net.minecraft.screen.slot.Slot;
+import net.minecraft.text.Text;
+
+public class HotmPerkLevelAdder extends SimpleSlotTextAdder {
+	private static final ConfigInformation CONFIG_INFORMATION = new ConfigInformation(
+			"hotm_perk_level",
+			"skyblocker.config.uiAndVisuals.slotText.hotmPerkLevel");
+	private static final Pattern LEVEL = Pattern.compile("Level (?<level>\\d+)\\/(?<max>\\d+)");
+
+	public HotmPerkLevelAdder() {
+		super("^Heart of the Mountain$", CONFIG_INFORMATION);
+	}
+
+	@Override
+	@NotNull
+	public List<SlotText> getText(@Nullable Slot slot, @NotNull ItemStack stack, int slotId) {
+		if (slotId >= 0 && slotId <= 53 && !stack.isOf(Items.COAL)) {
```

Comment:
There's no need for the `slotId` check to go beyond 44, the bottom row is always other stuff.
```suggestion
		if (slotId >= 0 && slotId <= 44 && !stack.isOf(Items.COAL)) {
```